types/core-js/index.d.ts

```diff
@@ -712,6 +712,7 @@ declare namespace core {
         from<T, U>(arrayLike: ArrayLike<T> | Iterable<T>, mapfn: (v: T, k: number) => U, thisArg?: any): U[];
         from<T>(arrayLike: ArrayLike<T> | Iterable<T>): T[];
         of<T>(...items: T[]): T[];
+        isArray(arg: any): arg is Array<any>;
```

Comment:
Hey @NN--- many thanks for your PR!

Why did you not declare a `boolean` returns directly?

Reply:
This allows type guard check.
E.g.
var abc:any;
if(core.Array.isArray(abc)){
// here abc is typed as any[] instead of just any.
}

Reply:
Great! @NN--- I understand now, this is a nice `ts` feature! 😄 Thanks a lot for your explanation 😄